src/Spryker/Zed/EventBehavior/Business/Model/EventEntityTransferFilter.php

```diff
@@ -145,6 +149,93 @@ public function getEventTransfersAdditionalValues(array $eventTransfers, string
         return array_unique($additionalValues);
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\HydrateEventsRequestTransfer $hydrateEventsRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\HydrateEventsResponseTransfer
+     */
+    public function hydrateEventDataTransfer(HydrateEventsRequestTransfer $hydrateEventsRequestTransfer): HydrateEventsResponseTransfer
```

Comment:
while we still here. How Filter class is related to hydration process

Reply:
it was requested from Rene